### PR TITLE
Major updates

### PR DESCRIPTION
**Pull request by selfagency**
- Changed semantic versioning to match version of Grid.js on which it is built to work
- Moved entirely to an ES6 module
- Dropped npm dependencies in favor of unpkg so it can be used entirely on the frontend without bundlers
- Removed inessential dependencies and lightened the weight of others
- Simplified the testing procedure

Please be sure to test before merging!

### CODE REVIEW

File: src/gridjs-vue.mjs [hunk 1]
```diff
@@ -0,0 +1,246 @@
+import { Grid } from 'https://unpkg.com/gridjs/dist/gridjs.production.es.min.js'
```
Comment (afshinm) [inline src/gridjs-vue.mjs:1]: Can you please update Grid.js to v5? I have changed the module format a little bit which can affect this, see https://unpkg.com/browse/gridjs@5.0.1/

File: src/index.mjs [hunk 2]
```diff
@@ -1,34 +1,25 @@
-import { createRef, h, html } from 'gridjs'
-import elementReady from 'element-ready'
-import { nanoid } from 'nanoid'
+import { createRef, h, html, PluginPosition } from 'https://unpkg.com/gridjs/dist/gridjs.production.es.min.js'
```
Comment (afshinm) [inline src/index.mjs:1]: Again please update Grid.js to v5

**Review by afshinm — COMMENTED** [review]
LGTM! :+1: I just released v5 please update Grid.js.

File: src/index.mjs (hunk 2 above)
Comment (selfagency) [reply src/index.mjs]: Done

File: src/gridjs-vue.mjs (hunk 1 above)
Comment (selfagency) [reply src/gridjs-vue.mjs]: Done

**Review by afshinm — APPROVED** [review]
:tada: :tada: :tada: 

LGTM!

**Pull request merged**